scripts/ledger.lic

```diff
@@ -98,6 +106,25 @@ module Ledger
     end
   end
 
+  module Utility
+    def self.get_note_value(noteId)
+      line = Lich::Util::quiet_command_xml("read ##{noteId}", /has a value of (?:[\d,]+) silver and reads/, /<prompt/, false, 3).last

```

Comment:
Thinking now this was a 5.7.0 thing so would delay release until out of beta (and possibly a while after to allow time to update). If this is a 5.7 thing is there a good alternative? Could pull the issue_command code directly into the script I guess.

Reply:
Yep, quiet_command_xml is the way to go for now until 5.7.0 is released. Can always change it later on once 5.7.0 has had time to bake in for awhile. Can also add a requires in the header and code at the top to warn if not on v5.7.0+ and exit out.